llvm/tools/llc/NewPMDriver.h

```diff
@@ -32,6 +32,8 @@ class ToolOutputFile;
 class LLVMContext;
 class MIRParser;
 
+enum class VerifierKind { NoVerifier, VerifyOut, VerifyEachPass };
```

Comment:
I thought the default was "verify input and output", so `VerifyOut` is misleading. `VerifyInputOutput`?

Ah I see that you copied this from opt. But yeah the name should be changed